read.h

```diff
@@ -54,6 +54,14 @@
 #define REDIS_REPLY_NIL 4
 #define REDIS_REPLY_STATUS 5
 #define REDIS_REPLY_ERROR 6
+#define REDIS_REPLY_DOUBLE 7
+#define REDIS_REPLY_BOOL 8
+#define REDIS_REPLY_VERB 9
```

Comment:
Do you know if `REDIS_REPLY_VERB` and `REDIS_REPLY_MAP` resolving to 9 intentional here?  Looks like it's in Redis proper too, but I don't see it referenced anywhere else.

Reply:
@michael-grunder I'd guess it's a mistake in the original commit.